docs/integrations/sources/amazon-ads.md

```diff
@@ -1,26 +1,33 @@
 # Amazon Ads
+
 This page contains the setup guide and reference information for the Amazon Ads source connector.
 
 ## Prerequisites
 
-* Client ID
-* Client Secret
-* Refresh Token
-* Region
-* Report Wait Timeout
-* Report Generation Maximum Retries
-* Start Date (Optional)
-* Profile IDs (Optional)
+- Client ID
```

Comment:
Is there a specific reason why we change the list syntax in all those files?

Reply:
This is Prettier default conf. hyphen and asterisk are the same in markdown

Reply:
Yes I know they are the same, but there is no setup that should run `prettier` on those files by default in the repository, so this might be a local setup in your IDE. We should in general not change a lot of unrelated syntax (like `*` into `-`) in a PR for something completely else. So We should make sure this PR is only touching adding the actual tags and changes related to it and not touching a lot of unrelated code in those files due to some local prettier setup.

Reply:
Do you now what can I do to reverse the prettier changes?